app/src/lib/stores/pull-request-store.ts

```diff
@@ -25,207 +25,264 @@ import { IRemote } from '../../models/remote'
  */
 export const ForkedRemotePrefix = 'github-desktop-'
 
+const Decrement = (n: number) => n - 1
+const Increment = (n: number) => n + 1
+
 /** The store for GitHub Pull Requests. */
 export class PullRequestStore extends TypedBaseStore<GitHubRepository> {
-  private readonly pullRequestDatabase: PullRequestDatabase
-  private readonly repositoriesStore: RepositoriesStore
-
-  private activeFetchCountPerRepository = new Map<number, number>()
+  private readonly _pullRequestDatabase: PullRequestDatabase
+  private readonly _repositoryStore: RepositoriesStore
+  private readonly _activeFetchCountPerRepository = new Map<number, number>()
 
   public constructor(
     db: PullRequestDatabase,
     repositoriesStore: RepositoriesStore
   ) {
     super()
 
-    this.pullRequestDatabase = db
-    this.repositoriesStore = repositoriesStore
+    this._pullRequestDatabase = db
+    this._repositoryStore = repositoriesStore
   }
 
   /** Loads all pull requests against the given repository. */
-  public async refreshPullRequests(
+  public async fetchPullRequests(
```

Comment:
Can we rename this method `fetchAndCachePullRequests` to indicate that it's also responsible for caching new pull requests?